token-metadata/interface/Cargo.toml

```diff
@@ -12,17 +12,26 @@ serde-traits = ["dep:serde", "spl-pod/serde-traits"]
 
 [dependencies]
 borsh = "1.5.1"
+num-derive = "0.4"
+num-traits = "0.2"
 serde = { version = "1.0.214", optional = true }
+solana-borsh = "2.1.0"
+solana-decode-error = "2.1.0"
+solana-instruction = "2.1.0"
+solana-msg = "2.1.0"
 solana-program = "2.1.0"
```

Comment:
Can we go with the same approach here and just re-export the crates that it uses? It looks like that's the only usage of solana-program in the crate, so we can remove it after that